.rspec

```diff
@@ -0,0 +1,2 @@
+--require spec_helper
+--require rails_helper
```

Comment:
At least you did not add the colors, I don't like it :joy: :joy: :joy: .

Thanks :)

Reply:
Yeah, the motivation was to avoid having to write require 'rails_helper' at the top of every file 😄